### PR TITLE
Refactor text_summarization for CI

### PR DESCRIPTION
**Pull request by gongel**
### PR types

Refactor
### PR changes

Applications
### Description

- Refactor text_summarization for CI

### CODE REVIEW

File: applications/text_summarization/pegasus/README.md [hunk 1]
```diff
@@ -133,6 +133,10 @@ data/
 "content": "“北京的保障房市场就像一个巨大的赌场，每个人都在期待中奖。”面对中国目前现行的保障性住房政策，华远地产董事长任志强再次语出惊人。（分享自@第一财经-中国房地产金融）"
 }
 ```
+这里提供小数据集供测试，运行下面命令即可下载:
+```bash
+bash run_prepare.sh
```
Comment (zjjlivein) [inline applications/text_summarization/pegasus/README.md]: python run_prepare.py

File: applications/text_summarization/pegasus/README.md [hunk 2]
```diff
@@ -144,14 +148,14 @@ data/
 # GPU启动，参数`--gpus`指定训练所用的GPU卡号，可以是单卡，也可以多卡
 unset CUDA_VISIBLE_DEVICES
 
-python -m paddle.distributed.launch --gpus "2,3,4,5,6,7" run_summarization.py \
+python -m paddle.distributed.launch --gpus "2,3,4,5,6,7" train.py \
     --model_name_or_path=IDEA-CCNL/Randeng-Pegasus-238M-Summary-Chinese \
     --train_file train.json \
     --eval_file test.json \
```
Comment (zjjlivein) [inline applications/text_summarization/pegasus/README.md]: data/train.json
data/test.json

File: applications/text_summarization/pegasus/README.md [hunk 3]
```diff
@@ -213,13 +214,12 @@ python -m paddle.distributed.launch --gpus "2,3,4,5,6,7" run_summarization.py \
 ```shell
 unset CUDA_VISIBLE_DEVICES
 
-python run_generate.py \
-    --model_name_or_path=pegesus_out/pegeaus_model_10000 \
+python predict.py \
+    --init_checkpoint_dir=pegesus_out \
     --prefict_file valid.json \
     --max_source_length 128 \
```
Comment (zjjlivein) [inline applications/text_summarization/pegasus/README.md:220]: data/valid.json

**Review by zjjlivein — DISMISSED** [review]

File: applications/text_summarization/pegasus/README.md (hunk 1 above)
Comment (gongel) [reply applications/text_summarization/pegasus/README.md]: DONE

File: applications/text_summarization/pegasus/README.md (hunk 2 above)
Comment (gongel) [reply applications/text_summarization/pegasus/README.md]: DONE

File: applications/text_summarization/pegasus/README.md (hunk 3 above)
Comment (gongel) [reply applications/text_summarization/pegasus/README.md:220]: DONE thx

**Comment by gongel** [comment]
Same as #4102

**Pull request closed without merging**